pennylane/qnode_new/decorator.py

```diff
@@ -87,12 +108,20 @@ def QNode(
     return node
 
 
-def qnode(device, *, interface="autograd", mutable=True, differentiable=True, properties=None):
+def qnode(device, *, interface="autograd", mutable=True, diff="best", properties=None):
     """Decorator for creating QNodes.
 
     When applied to a quantum function, this decorator converts it into
     a :class:`QNode` instance.
 
+    **Example:**
+
+    >>> dev = qml.device("default.qubit", wires=1)
+    >>> @qnode(dev)

```

Comment:
How would ``qnode`` be accessed after the refactor? Would it not be imported in ``pennylane/__init__.py``? If It would be, ``@qml.qnode(dev)`` would still be needed here, correct?

Reply:
Yep, this is a typo and should be fixed! Good catch
```suggestion
    >>> @qml.qnode(dev)
```